.github/workflows/release.yml

```
@@ -0,0 +1,167 @@
+name: Release
+
+env:
+  GO111MODULE: "on"
+  GOPATH: "/home/runner/work/armada/armada/go"
+  GOCACHE: "/home/runner/.cache/go-build"
+  GOVERSION: "1.16"
+
+on: 
+  push:
+    tags: 
+      - 'v*'
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    env:
+      DOCKERHUB_PASS: ${{ secrets.DOCKERTOKEN }}
+      DOCKERHUB_USER: ${{ secrets.DOCKERHUB_USER }}
+    steps:
+      - name: Checkout code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          path: ${{ env.GOPATH }}/src/github.com/G-Research/armada
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GOVERSION }}
+
+      - name: Build armadactl release artifacts
+        run: |
+          TAG=${GITHUB_REF##*/}
+          make build-armadactl-release RELEASE_VERSION=${TAG}
```

@ljubon i would remove the line `TAG=${GITHUB_REF##*/}` and just do something like:
```suggestion
          - name: Build armadactl release artifacts
            run: make build-armadactl-release RELEASE_VERSION=${GITHUB_REF##*/}
            working-directory: ${{ env.GOPATH }}/src/github.com/G-Research/armada
```